certified-connectors/Kapish Power Connect/apiProperties.json

```diff
@@ -0,0 +1,290 @@
+                "tooltip": "Provide the Client ID for authentication",
+                "constraints": {
+                  "tabIndex": 5,
+                  "required": "true"
+                }
+              }
+            },
+            "clientSecret": {
+              "type": "securestring",
+              "uiDefinition": {
+                "displayName": "Client Secret",
+                "description": "The Client Secret for authentication",
+                "tooltip": "Provide the Client Secret for authentication",
+                "constraints": {
+                  "tabIndex": 6,
+                  "required": "true",
+                  "clearText": false
+                }
+              }
+            }
+          }
+        },
+        {
+          "name": "Basic",
+          "uiDefinition": {
+            "displayName": "System Managed",
+            "description": "Use a user name and password to connect to Content Manager"
+          },
+          "parameters": {
+            "apiKey": {
+              "type": "securestring",
+              "uiDefinition": {
+                "displayName": "KPC Subscription Key",
+                "description": "The API key for accessing Kapish Power Connect",
+                "tooltip": "Provide the API key",
+                "constraints": {
+                  "tabIndex": 1,
+                  "required": "true",
+                  "clearText": false
+                }
+              }
+            },
+            "cmServiceApiUrl": {
+              "type": "string",
+              "uiDefinition": {
+                "displayName": "CM ServiceAPI URL (e.g. https://myserver/CMServiceAPI/)",
+                "description": "The URL where your CM ServiceAPI can be reached",
+                "tooltip": "Provide the CM Service API URL",
+                "constraints": {
+                  "tabIndex": 2,
+                  "required": "true"
+                }
+              }
+            },
+            "userName": {
+              "type": "string",
+              "uiDefinition": {
+                "displayName": "User Name",
+                "description": "The user name for authentication",
+                "tooltip": "Provide the Client ID for authentication",
+                "constraints": {
+                  "tabIndex": 3,
+                  "required": "true"
+                }
+              }
+            },
+            "password": {
+              "type": "securestring",
+              "uiDefinition": {
+                "displayName": "Password",
+                "description": "The password for authentication",
+                "tooltip": "Provide the password for authentication",
+                "constraints": {
+                  "tabIndex": 4,
+                  "required": "true",
+                  "clearText": false
+                }
+              }
+            }
+          }
+        }
+      ]
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@BrettHankinson please make sure you are using the iconBrandColor for your connector. Icon color must contain a non-transparent, non-white color (#ffffff) background and not-default color (#007ee5) that matches your specified icon background color. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).